clusters/app.ci/vault/manifests.yaml

```diff
@@ -0,0 +1,175 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: vault
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: cert-issuer
+    route.openshift.io/termination: passthrough
```

Comment:
Aren't this and the TLS configuration mutually exclusive?

Reply:
No. The tls config tells the ingress that this has TLS. This tells the route controller that actually implements this that it should not touch tls traffic and just pass it through to the backend.

Reply:
But what is the meaning of `secretName` if traffic is passed through to the pod?  Or is it just a requirement of the API?

Reply:
It is not mandatory from the API perspective but we need to set it so cert-manager knows where to put the cert it gets from lets encrypt